joomla/joomla-cms
# [5.1] Fix content history diff error

### Summary of Changes

This PR fixes a JS error when trying to view the "diff" between content versions, due the dependency not being called.

### Testing Instructions

1. Ensure "Versions" is enabled
2. Create an article with some dummy content and save it
3. Add or remove some content to the same article as save it
4. Compare the two versions of the article

### Actual result BEFORE applying this Pull Request

> Uncaught TypeError: window.Diff is undefined

![before](https://github.com/joomla/joomla-cms/assets/2019801/452d1b73-3570-443d-bc30-734be9a5fbef)

### Expected result AFTER applying this Pull Request

The diff is displayed, showing green/red blocks

![after](https://github.com/joomla/joomla-cms/assets/2019801/df2926df-fa23-4d03-ac62-3f4a881dfb62)

### Note

I'm aware the diff fails a11y standards for dark mode, but this should be fixed in a separate PR
